composer.json

```diff
@@ -50,7 +51,8 @@
         "psr-4": {
             "App\\": "app/",
             "Database\\Factories\\": "database/factories/",
-            "Database\\Seeders\\": "database/seeders/"
+            "Database\\Seeders\\": "database/seeders/",
+            "Modules\\": "app/Modules"
```

Comment:
Why was this added?

Reply:
I was testing things and forgot to remove it. I force pushed an update that removes this.